cypress/support/commands.js

```diff
@@ -23,3 +23,36 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+// Custom command that bundles the iframe method and types text into it 
+Cypress.Commands.add('typeInIframeElement', (elementSelector, textToType) => {
+  return cy
+    .iframe('[id="paymentsParentDivIdIframe"]', elementSelector).as('checkoutiFrame')
+    .then($iframe => {
+      cy.get('@checkoutiFrame')
+      .find(elementSelector).first()    
+      .type(textToType, {force: true}) 
+    })
+})
+
+// Cypress has no built in command to handle iframes, this is a custom command to find and move into an element in an iframe
+Cypress.Commands.add('iframe', (iframeSelector, elSelector) => {
+    return cy
+      .get(`iframe${iframeSelector || ''}`, { timeout: 10000 })
+      .should($iframe => {
+        expect($iframe.contents().find(elSelector||'body')).to.exist
+      })
+      .then($iframe => {
+        return cy.wrap($iframe.contents().find('body'))
+      })
+  })
+
+  // Cypress has no tab command, was thinking of using this to tab to next input box to show error message
+  Cypress.Commands.add('typeTab', (shiftKey, ctrlKey) => {
```

Comment:
This is a cleaver workaround.  You can also use built-in methods like `blur()` to trigger this behavior.